app/controllers/my-services/get-index.controller.js

```diff
@@ -8,7 +8,7 @@ const serviceService = require('../../services/service.service')
 const { filterGatewayAccountIds } = require('../../utils/permissions')
 const getHeldPermissions = require('../../utils/get-held-permissions')
 const { DEFAULT_SERVICE_NAME } = require('../../utils/constants')
-const showNewContractTermsBannerUntilDate = process.env.SHOW_NEW_CONTRACTS_BANNER_UNTIL_DATE || '1645488000'
+const showNewContractTermsBannerUntilDate = process.env.SHOW_NEW_CONTRACTS_BANNER_UNTIL_DATE || '1646006400'
```

Comment:
As per story this should be until `21-Mar-22`

Reply:
Good spot - just updated it.